src/website/app/demos/Table/examples/sortable.js

```diff
@@ -0,0 +1,63 @@
+/* @flow */
+import Table from '../../../../../library/Table';
+import sharedData from '../shared/data';
+
+export default {
+  id: 'sortable',
+  title: 'Sortable Columns',
+  description: `Users can sort the rows in Table by column, enabled via the
+\`sortable\` prop. If the default sorting function, below, is insufficient for
+your needs, you can supply your own with the \`sortComparator\` prop. You can
+set the initially sorted column & direction with the \`defaultSort\` prop. An
+\`onSort\` callback is also available. Note that the \`sortable\` &
+\`sortComparator\` properties can be applied to Table via props or to individual
+columns via [column definition](/components/table/#Column-type).
+
+\`\`\`
+// Coerce null & undefined values to an empty string and normalize letter casing
+const normalizedValue = (value) =>
+  value === null || value === undefined
+    ? ''
+    : typeof value === 'string' ? value.toUpperCase() : value;
+
+const defaultSortComparator: SortComparator = (a, b, key) => {
+  const valueA = normalizedValue(a[key]);
+  const valueB = normalizedValue(b[key]);
+
+  return valueA < valueB ? -1 : valueA > valueB ? 1 : 0;
+};
+\`\`\`
+
+In the example below, all columns except the last one have enabled sorting and
+the Dairy column has a custom sorting function to sort by length rather than
+alphabetically.
```

Comment:
Small wording change; replace "...all columns except the last one have enabled sorting and the Dairy column has a custom sorting function to sort by length rather than alphabetically." with something like "...sorting is enabled in all columns except for the "Protein" one. The sorting function in the "Dairy" column is custom-built to sort by length rather than alphabetically"